content/en/docs/components/serving/kfserving.md

```diff
@@ -74,7 +74,7 @@ Kubeflow 1.1 includes KFServing v0.3, where the focus has been on providing more
 
 ### Sample notebooks
 
-* [SDK client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client/kfserving_sdk_sample.ipynb)
+* [SDK client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client)
```

Comment:
```suggestion
* [SDK client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client/kfserving_sdk_v1alpha2_sample.ipynb)
```

Reply:
@animeshsingh I had it on purpose on the directory level, since there are now 2 notebooks.
By using the directory, the link does not need to change if the underlying notebook(s) change again.
I did the same in the readme on kfserving kubeflow/kfserving#1193 which was approved by @yuzisun

Reply:
hear you - but the website docs point to an official release, and that section is title "notebooks" specifically. The 2nd notebook is still not official for overall Kubeflow

Reply:
ok. Clear. But should I then not also change the same link in the SDK Section?

Reply:
it's fine there. no strong opinion